transfer.md

```diff
@@ -63,54 +69,64 @@ There are other equivalents for these tools
 | `docker system info`   | [`podman system info`](./docs/source/markdown/podman-system-info.1.md)      |
 | `docker system prune`  | [`podman system prune`](./docs/source/markdown/podman-system-prune.1.md)    |
-| `docker system`        | [`podman system`](./docs/source/markdown/podman-system.1.md)                |
 | `docker tag`     | [`podman tag`](./docs/source/markdown/podman-tag.1.md)          |
 | `docker top`     | [`podman top`](./docs/source/markdown/podman-top.1.md)          |
 | `docker unpause` | [`podman unpause`](./docs/source/markdown/podman-unpause.1.md)  |
 | `docker version` | [`podman version`](./docs/source/markdown/podman-version.1.md)  |
+| `docker volume        `| [`podman volume`](./docs/source/markdown/podman-volume.1.md)		       |
 | `docker volume create` | [`podman volume create`](./docs/source/markdown/podman-volume-create.1.md)  |
 | `docker volume inspect`| [`podman volume inspect`](./docs/source/markdown/podman-volume-inspect.1.md)|
 | `docker volume ls`     | [`podman volume ls`](./docs/source/markdown/podman-volume-ls.1.md)          |
 | `docker volume prune`  | [`podman volume prune`](./docs/source/markdown/podman-volume-prune.1.md)    |
 | `docker volume rm`     | [`podman volume rm`](./docs/source/markdown/podman-volume-rm.1.md)          |
-| `docker volume`  | [`podman volume`](./docs/source/markdown/podman-volume.1.md)			|
 | `docker wait`          | [`podman wait`](./docs/source/markdown/podman-wait.1.md)		       |
 
-**** Use mount to take advantage of the entire linux tool chain rather then just cp.  Read [`here`](./docs/podman-cp.1.md) for more information.
-
 ## Behavioural differences and pitfalls
 
 These commands behave differently from the commands in Docker:
 
 | Command | Description |
 | :--- | :--- |
-| `podman volume create` | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) for more information.|
+| `podman volume create`                | While `docker volume create` is idempotent, `podman volume create` raises an error if the volume does already exist. See this [docker issue](https://github.com/moby/moby/issues/16068) for more information.|
+| `podman run -v /tmp/noexist:/tmp ...` | While `docker run -v /tmp/noexist:/tmp` will create non existing volumes on the host, Podman will report that the volume does not exist. The Podman team sees this as a bug in Docker.|
 
 ## Missing commands in podman
 
 Those Docker commands currently do not have equivalents in `podman`:
 
 | Missing command | Description|
 | :--- | :--- |
-| `docker container rename`   | podman does not support `container rename` - or the `rename` shorthand. We recommend using `podman rm` and  `podman create` to create a container with a specific name.|
+| `docker builder`  ||
+| `docker build*`   ||
```

Comment:
I think we have this one?

Reply:
Did you mean `buildx`?

Reply:
Yup, docker --help refers to this as buildx*, some how we dropped the x, now I am dropping the *